src/basilisp/core.lpy

```diff
@@ -26,7 +26,7 @@
 (def ^{:doc      "Create a list from the arguments."
        :arglists '([& args])}
   list
-  (fn* list [& args] args))
+  (fn* list [& args] (if args args '())))
```

Comment:
This was previously relying on the fact that `args` would just _be_ a list. Now that it might be `nil` we need to return the empty list if no args are given.